src/test/resources/regressions/features/opaque/opaque-interface-fail1.gobra

```diff
@@ -0,0 +1,23 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+// This does not parse:
+// type foo interface {
+//   opaque pure bar() (res int)
+// }
+
+// QUES: This both parses and type checks. I am not sure why.
```

Comment:
As the comment says, I am not exactly sure why this both parses and type checks. I am also unsure on how we want to handle interfaces.

Reply:
Why do you think this should be rejected?

Regarding interfaces, I would have a type check that ensures that interface method signatures do not have `opaque`. The reason is that `opaque` is not meaningful here. Unlike with an abstract function, we will never be able to modify the code such that the interface method now has a body. Furthermore, it may confuse users who wonder how the `opaque` annotation interferes with behavioral subtyping.